src/routes/deepLink/__tests__/deepLinkJoinCommunityAuthenticated.tsx

```diff
@@ -76,14 +78,8 @@ describe('JoinGroupScreen next', () => {
     });
 
     expect(store.getActions()).toEqual([
-      {
-        analyticsContext: {
-          [ANALYTICS_PREVIOUS_SCREEN_NAME]: 'mh : deep link : community',
-        },
-        type: ANALYTICS_CONTEXT_CHANGED,
-      },
```

Comment:
So I'm really confused about this. There's a ton of test that have changes similar to this. Should this change? My changes shouldn't have affected this, at least I think. I didn't fix all the tests yet just in case something was wrong.

Reply:
I assume `FBAnalytics().logScreenView` is undefined which throws an error when you try to call it. You've mocked `trackScreenChange` in this file so it no longer calls it but now you have to setup and check for a mock value.